docs/installation_guide.md

```diff
@@ -277,6 +277,47 @@ Please check the [Docker Compose command-line reference](https://docs.docker.com
 By default, registry data is persisted in the target host's `/data/` directory.  This data remains unchanged even when Harbor's containers are removed and/or recreated.
 In addition, Harbor uses `rsyslog` to collect the logs of each container. By default, these log files are stored in the directory `/var/log/harbor/` on the target host.  
 
+## Configuring Harbor listening on a customized port
+By default, Harbor listens on port 80 both for admin portal and docker commands, you can customize the port by following steps:  
+
+1.Modify Deploy/docker-compose.yml  
+Replace the first "80"(HTTP protocal) to a customized port 8888 for example. If HTTPS is needed, replace the first "443".  
+
+```
+proxy:
+    image: library/nginx:1.9
+    restart: always
+    volumes:
+      - ./config/nginx:/etc/nginx
+    ports:
+      - 8888:80
+      - 443:443
+    depends_on:
+      - mysql
+      - registry
+      - ui
+      - log
+    logging:
+      driver: "syslog"
+      options:  
+        syslog-address: "tcp://127.0.0.1:1514"
+        tag: "proxy"
+```
+
+2.Modify Deploy/templates/registry/config.yml  
+Add the ":8888" after "$ui_url".  
+
+```  
+auth:
+  token:
+    issuer: registry-token-issuer
+    realm: $ui_url:8888/service/token
+    rootcertbundle: /etc/registry/root.crt
+    service: token-service
+```
+
+3.Execute Deploy/prepare and start Harbor.
```

Comment:
Execute Deploy/prepare script and restart Harbor. consider to give the commands.

Do we need compose down here?